bsp/simulator/rtconfig_project.h

```diff
@@ -14,6 +14,7 @@
 
 #ifdef _MSC_VER
 /* disable some warning in MSC */
+#pragma warning(disable:4103)   /* structure packing changed by including file */
```

Comment:
```suggestion
// #pragma warning(disable:4103)   /* structure packing changed by including file */
```
建议改成这种方式，可以提示但是不要直接disable掉，因为这个并没有实际解决问题，只是在自欺欺人。